wall_e_models/models.py

```diff
@@ -30,10 +31,14 @@ class BanRecord(models.Model):
     mod_id = models.BigIntegerField(null=True)
     ban_date = models.BigIntegerField(null=True)
     reason = models.CharField(max_length=512, null=False)
-    unban_date = models.BigIntegerField(null=True, default=None)
+    unban_date = models.BigIntegerField(null=False, default = 0)
 
     class Meta:
         db_table = 'wall_e_models_ban_records'
+        constraints = [
+            UniqueConstraint(fields=['user_id', 'unban_date'],
+                             name='unique_active_ban')
+        ]
```

Comment:
```suggestion
        constraints = [
            UniqueConstraint(fields=['user_id'], name='unique_active_ban', condition=Q(unban_date__isnull=True))
        ]
```

doing this allow `unban_date` to remain `None` which is a better idea